requirements.txt

```diff
@@ -1,2 +1,2 @@
-flywheel-sdk==10.0.1
+flywheel-sdk<=11.0.0
```

Comment:
Thinking `~=11.0.0` is better (not sure what got fixed but they are already at 11.0.1)